modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx

```diff
@@ -227,9 +227,19 @@ const ModalAddObjectLayoutTab: React.FC<IModalAddObjectLayoutTabProps> = ({
 							contentRight={
 								<ClayLabel
 									className="label-inside-custom-select"
-									displayType="secondary"
+									displayType={
```

Comment:
@danvb112 I think we have a lot of the structure "ternary inside ternary" and this could harm the legibility of the code. You could do something like:
![2022-08-17_14-18](https://user-images.githubusercontent.com/60627148/185203111-c59bb39e-6be3-48d9-9082-fa17dd244d12.png)

and then call:
![2022-08-17_14-19](https://user-images.githubusercontent.com/60627148/185203173-1e08648b-9041-4326-882a-bc53c2069070.png)

You can also do the same thing for the other cases in this file. What do you think?

Reply:
Thanks for the suggestion @naraandrade! I made the changes